Jenkinsfile_CNP

```diff
@@ -9,6 +9,13 @@ def component = "user-profile-api"
 def secrets = [
         's2s-${env}': [
                 secret('microservicekey-rd-user-profile-api', 'S2S_SECRET')
+        ],
+        'rd-${env}': [
+                secret('user-profile-api-POSTGRES-HOST', 'FUNC_DATABASE_HOST'),
+                secret('user-profile-api-POSTGRES-PASS', 'FUNC_DATABASE_PASS'),
+                secret('user-profile-api-POSTGRES-USER', 'FUNC_DATABASE_USER'),
```

Comment:
add the entries in the jenkins_nightly file

Reply:
Earlier it was decided to run script from only master/aat build, now it will run only in nightly.